cafe/engine/models/data_interfaces.py

```diff
@@ -140,7 +140,7 @@ class ConfigParserDataSource(DataSource):
     def __init__(self, config_file_path, section_name):
         super(ConfigParserDataSource, self).__init__()
 
-        cafe_env_var = {key: value for key, value in os.environ.iteritems()
```

Comment:
I appreciate the feedback, but his change should be here; The increased coverage provided by new unittest associated with this PR uncover this bug. If we do not address it, the PR will fail gate jobs. Additionally, this follows the boy scout rule of leaving the code better than we found it.